setup.cfg

```diff
@@ -42,6 +42,8 @@ install_requires =
     notebook_shim>=0.1
     jinja2>=3.0.3
     y-py>=0.4.6,<0.5.0
+    ypy-websocket>=0.1.2
+    jupyter_ydoc>=0.1.0
```

Comment:
The question about where that code should live is good. But I'm reluctant to multiply the downstream dependencies here. I would prefer to keep them in this repo or move them to `jupyterlab_server` for now if the goal is to allow reuse without having to link to the big `jupyterlab` package.

And whatever is the final choice, that code must be part of one Jupyter GitHub organization.

Reply:
The plan is to move:
- `ypy-websocket` to https://github.com/y-crdt,
- `jupyter_ydoc` to https://github.com/jupyter-server.

Reply:
We can move `jupyter_ydoc` to `jupyter-server`, but is it an issue that `ypy-websocket` is not part of a Jupyter GH org? It's the same for `y-py`, these libraries are not tied to Jupyter IMO.

Reply:
y-crdt is definitely a good candidate too.